### ISSUE TITLE
Need to change a line of code in `frn_layer.py` for Pytorch Version==1.0.0 

### ISSUE THREAD

**Issue by Huixxi** [knjcode/filter-response-normalization-layer-pytorch#2]
Change
```python
 nu2 = torch.mean(input**2, axis=[2, 3], keepdims=True)
```
to
```python
 nu2 = torch.mean(input**2, dim=(2, 3), keepdim=True, out=None)
```

**Comment by knjcode** [comment]
Thanks for pointing out!
I'll fix the code later.
